### DISCUSSION TITLE
DownloadHelper cancellation and refund request

### DISCUSSION THREAD

**Discussion by Gicivan** [aclap-dev/video-downloadhelper#60] [License & Purchase requests]
### What's your issue?

DownloadHelper as an extension of Google Chromes is not serving me the way I would like it to be. However, that extension, when used in Firefox, works well and performs properly the downloads I need.

I would very much like the DownloadHelper company to grant me the cancellation of the Google Chromes license and the consequent refund of the amount paid ($ 14.99 USD) because I wish to purchase the unlimited Premium license through the Firefox browser. 

### The 10 first digits (only 10) of your license number or email address

7cc03d6767

### I confirm that I didn't not paste my full license number

- [X] I confirm

**Comment by paulrouget** [comment]
See license problem here: https://github.com/aclap-dev/video-downloadhelper/wiki/I-need-help#license-problem - there's an email you message.
